hawkular-inventory-json-helper/src/main/java/org/hawkular/inventory/json/mixins/DataEntityMixin.java

```diff
@@ -19,20 +19,26 @@
 import java.util.Map;
 
 import org.hawkular.inventory.api.model.CanonicalPath;
+import org.hawkular.inventory.api.model.DataEntity;
 import org.hawkular.inventory.api.model.StructuredData;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
  * @author Lukas Krejci
  * @since 0.3.0
  */
-public class DataEntityMixin {
+public abstract class DataEntityMixin {
 
     @JsonCreator
     public DataEntityMixin(@JsonProperty("path") CanonicalPath path, @JsonProperty("value") StructuredData value,
             @JsonProperty("properties") Map<String, Object> properties) {
     }
 
+    @JsonIgnore abstract String getId();
```

Comment:
Without this change the `DataEntity` JSON looks like: 
```json
{
  "path":"/e;e/r;r/d;connectionConfiguration",
  "value":[1,2],
  "properties":{},
  "role":"connectionConfiguration",
  "id":"connectionConfiguration"
}
```

=> a lot of redundancy
With this change it looks like:
```json
{
  "path":"/e;e/r;r/d;connectionConfiguration",
  "value":[1,2],
  "properties":{}
}
```

It's more similar to the `DataEntity.{Blueprint|Update}` JSON which are inputs to the modifying ops and contains only the `StructuredData`, i.e. the their JSON looks like:
```json
{
  "value":[1,2],
}
```
(the `DataEntity.Update` contains also the properties)